test/jdk/TEST.ROOT

```diff
@@ -31,7 +31,9 @@ exclusiveAccess.dirs=java/math/BigInteger/largeMemory \
 java/rmi/Naming java/util/prefs sun/management/jmxremote \
 sun/tools/jstatd sun/security/mscapi java/util/Arrays/largeMemory \
 java/util/BitSet/stream javax/rmi java/net/httpclient/websocket \
-com/sun/net/httpserver/simpleserver
+com/sun/net/httpserver/simpleserver \
+java/awt \
+javax/swing
```

Comment:
Hello Harshitha, I don't have any knowledge of the client area, but I see that there are a large number of tests that are present in `test/jdk/java/awt` and `test/jdk/javax/swing`. Adding these 2 directories to `exclusiveAccess.dirs` would mean that none of the tests in `java/awt` will be run when any other test in `java/awt` is currently in progress. Same with the `javax/swing` directory. So it could increase the duration of the tiers in which these tests run. Is there a CI run which shows if this changes the test run duration drastically?

Reply:
@jaikiran Thank you for the details. I'll check if the duration of run differed drastically. I thought all the client tests run in non-concurrent mode by default or does this differ for headful and headless test?

Reply:
*Headful* tests are not supposed concurrently: they display UI and often use Robot to send input events to the UI. Running such tests concurrently leads to obscure failures. In our CI system, `JTREG_JOBS=1` is passed to `make` when running headful tests.

There's a set of *headless* tests which can be run concurrently. Such tests will be affected if they're run concurrently.

Reply:
@aivanov-jdk If it affects the run time of certain group of tests then it is better not to add it to `exclusiveAccess.dirs` and explicitly specify the `JTREG_JOBS=1` instead as both produce same results?

Reply:
As discussed, *headless* suite is affected. Therefore neither `java/awt` nor `javax/swing` should be added to `exclusiveAccess.dirs`.